src/cmd/linuxkit/run_vcenter.go

```diff
@@ -43,21 +43,21 @@ func runVcenter(args []string) {
 	flags := flag.NewFlagSet("vCenter", flag.ExitOnError)
 	invoked := filepath.Base(os.Args[0])
 
-	newVM.vCenterURL = flags.String("url", os.Getenv("VCURL"), "URL in the format of https://username:password@host/sdk")
-	newVM.dsName = flags.String("datastore", os.Getenv("VMDATASTORE"), "The Name of the DataStore to host the VM")
-	newVM.networkName = flags.String("network", os.Getenv("VMNETWORK"), "The VMware vSwitch the VM will use")
-	newVM.vSphereHost = flags.String("hostname", os.Getenv("VMHOST"), "The Server that will run the VM")
+	newVM.vCenterURL = flags.String("url", os.Getenv("VCURL"), "URL of VMware vCenter in the format of https://username:password@VCaddress/sdk")
+	newVM.dsName = flags.String("datastore", os.Getenv("VCDATASTORE"), "The name of the DataStore to host the VM")
+	newVM.networkName = flags.String("network", os.Getenv("VCNETWORK"), "The network label the VM will use")
+	newVM.vSphereHost = flags.String("hostname", os.Getenv("VCHOST"), "The server that will run the VM")
 
-	newVM.vmName = flags.String("vmname", "", "Specify a name for virtual Machine")
+	newVM.vmFolder = flags.String("vmfolder", "", "Specify a name/folder for the virtual machine to reside in")
 	newVM.path = flags.String("path", "", "Path to a specific image")
 	newVM.persistent = flags.String("persistentSize", "", "Size in MB of persistent storage to allocate to the VM")
 	newVM.mem = flags.Int64("mem", 1024, "Size in MB of memory to allocate to the VM")
 	newVM.vCpus = flags.Int("cpus", 1, "Amount of vCPUs to allocate to the VM")
 	newVM.poweron = flags.Bool("powerOn", false, "Power On the new VM once it has been created")
 
 	flags.Usage = func() {
-		fmt.Printf("USAGE: %s run vcenter [options] [name]\n\n", invoked)
-		fmt.Printf("'name' specifies the full path of an image that will be ran\n")
+		fmt.Printf("USAGE: %s run vcenter [options] path\n\n", invoked)
+		fmt.Printf("'path' specifies the full path of an image that will be ran\n")
```

Comment:
Updating, i've used "to run" below so changing it keeps the language consistent.